graph/graph-options.go

```diff
@@ -104,6 +106,11 @@ func FlagsToOptions() (graphOptions GraphOptions) {
 		tsInterval = uint64(*dEdgePtr)
 	}
 	deleteOnExpire := (24 * 60 * 60) * uint64(*windowPtr)
+	if *skipDelProbPtr != 0 {
+		if *skipDelProbPtr < 0 || *skipDelProbPtr >= 1 {
+			log.Panic().Msg("sdp must be in the range [0, 1)")
```

Comment:
```suggestion
			log.Panic().Msg("Skip deletion probability must be in the range [0, 1)")
```

Reply:
Thanks! Actually I think this option is still not ready yet because its performance is too bad (due to the linear search). Maybe we should hide this command line option. The max-flow algorithm can now handle deletes efficiently so a simple sliding window graph should be fine for the evaluation.